pkgs/tools/security/pcsclite/default.nix

```diff
@@ -10,6 +10,7 @@
 , polkit
 , systemdMinimal
 , IOKit
+, polkitSupport ? false
```

Comment:
I'm no fan of polkit, but having this default to `false` is a rather major user-visible change, no?

Your PR, minus this latest commit, was looking really good and uncontroversial.  Why not leave it at that, and move the defaults-changing/package-adding into a separate subsequent PR?

Reply:
True, but the fact is polkit support makes pcscd unusable if you don't have polkit running.
The polkit support was added fairly recently (2b93e96d0bdf5cd98c6d7bd60af6e4dded4ab3d1) and it only caused trouble as far as I can tell: in addition to this issue it also formed a dependency cycle with dbus...